### ISSUE TITLE
Free space in the docs server

### ISSUE THREAD

**Issue by datapythonista** [pandas-dev/pandas#45944]
I deleted an old backup of the docs, and we should be ok for a bit longer with the error when synching the development docs (see #45690)

I've been having a look, and there are couple of things that need to be done. Remove old kernel files, and check the rotation of the nginx logs.

Those are the directories taking more space. Our docs are around 15 Gb and are in `/usr/share`.
```
1.9G	boot
4.0G	lib
25G	usr
7.3G	var
```

The main directories taking a lot of space are next:
```
/var/log/nginx/access.log (6 Gb approx, seems like this needs to be rotated (zipped and start a new log file)
Eveything in /boot (kernel binaries), /usr/src (kernel headers) and /bin/modules/ (kernel modules) except for current kernel version (14 Gb approx)
```

I need to have a look and see exactly about these kernel files, but I think removing the old versions, we should avoid having problems. And also, not sure when logs are rotated, but I think we should zip them before they grow to Gbs. I'll wait a bit in case there are comments, and I'll move forward with this in few days.

**Comment by rhshadrach** [comment]
I'm seeing recent builds still failing, e.g https://github.com/pandas-dev/pandas/runs/5188152964

**Comment by mroeschke** [comment]
Just another example of a recent failure https://github.com/pandas-dev/pandas/runs/5193875376?check_suite_focus=true

**Comment by datapythonista** [comment]
I'm on it. It's a bit tricky, since most operations I need to perform to clean up fail because of lack of disk space, but I should have it working again in the next few hours.

**Comment by datapythonista** [comment]
I removed old kernel files, and compressed the logs and we've got 21Gb free now. Seems like the log compression isn't working, I think that's the reason we had the problem repeating quickly. Everything seems to be fine with it, and was working until one month ago, not sure if the first time the disk was full the rotation stopped working for some reason. I restarted the server after the fix, so hopefully everything should be working now. I'll try to have a look later this week, and see if the logs are being rotated now.
